frontend/src/components/NftDropCardStructuredList.js

```diff
@@ -49,9 +49,8 @@ export default function NftDropCardStructuredList({ nftDataArray }) {
 			orderType = "DESC";
 			break;
 		case "hot":
-			// TODO: order by left NFTs
-			orderField = "drop_id";
-			orderType = "ASC";
+			orderField = "drop_reserved";
```

Comment:
Well, maybe the "hot" ordering should be based on a normalized value, otherwise a drop with 3/10000 would be hotter then a limited drop with 2/5 NFTs already reserved

Reply:
My original idea of "hot" was the current "speed" of sold NFTs. It's difficult to compute a value without having the time therefore I proposed a "hot" similar to eBays "only X left", but I admit, the idea is not perfect :-/ .

Reply:
nothing against this idea, the "problem" here is only that isn't ordered by percent but by absolute counts